cli/test/01-cli.js

```diff
@@ -1,8 +1,10 @@
-/* eslint-env node, mocha */
-'use strict';
+/* eslint-env node */
```

Comment:
No need for using `require` since `ava` transpiles your test files so you can use `module` syntax.

Reply:
Yeah, good catch! I was trying `testdouble` for stubbing before switching to `ninos`, and this required to do something like this:

```js
const td = require('testdouble');

td.replace('npm-profile', mock);

const login = require('../lib/commands/login');
```

... which is not possible with static imports. I'll refactor it back to static ones, seems like this is something that is a common practice when using ava